SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -413,16 +416,23 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 		Connection->GetCoordinator().CreateSubView(SpatialConstants::ACTOR_AUTH_TAG_COMPONENT_ID, ActorFilter, RefreshCallbacks);
 	const SpatialGDK::FSubView& ActorNonAuthSubview =
 		Connection->GetCoordinator().CreateSubView(SpatialConstants::ACTOR_NON_AUTH_TAG_COMPONENT_ID, ActorFilter, RefreshCallbacks);
+	const SpatialGDK::FSubView& TombstoneActorSubview = Connection->GetCoordinator().CreateSubView(
+		SpatialConstants::TOMBSTONE_TAG_COMPONENT_ID, SpatialGDK::FSubView::NoFilter, SpatialGDK::FSubView::NoDispatcherCallbacks);
+	DebugActorSubView =
```

Comment:
What's the benefit of the netdriver maintain a reference to the debug actor subview? I'd rather the debug ctx subview owns it, similar to how the other subviews function.

Reply:
good catch- it shouldn't be a member of the net driver

Reply:
it turns out this is actually horrific

Reply:
debug context is initialised by passing a static function the net driver and this appears to be able to happen multiple times- when it succeeds the static function sets a field of the net driver to be an instance of the debug context which is also instantiated inside the static function (???) because otherwise it is possible to enable debugging too soon?

Reply:
my current plan is to rejig this such that: 
- the debug context is created with all the other core classes (if the world settings allows - if anyone knows why the world settings can't be accessed there let me know)
- the subview is passed in at this point if it debug is enabled
- the context is _initialised_ during startup to make sure the LB strategy is ready if it was created (if not nullptr etc.)

Reply:
I did this, and no, the world settings can't be accessed as the world isn't initialised at this point.

Reply:
alright it now owns the subview, but there is other follow on work that needs to be done here- I didn't realise it was using a non-tag component for the subview query tag, and should probably use the actor non-auth tag with a filter for the gdk debug component, because otherwise authority over the debug component might result in checking out an entity before the whole query result is received. It's such an awkward system that I think that is for a future PR